README.md

```diff
@@ -1,41 +1,53 @@
-aw-webui
-========
+# aw-webui
 
 > A webui for ActivityWatch built in Vue.js
 
 [![Build Status](https://travis-ci.org/ActivityWatch/aw-webui.svg?branch=master)](https://travis-ci.org/ActivityWatch/aw-webui)
-[![Known Vulnerabilities](https://snyk.io/test/github/ActivityWatch/aw-webui/badge.svg)](https://snyk.io/test/github/ActivityWatch/aw-webui) 
+[![Known Vulnerabilities](https://snyk.io/test/github/ActivityWatch/aw-webui/badge.svg)](https://snyk.io/test/github/ActivityWatch/aw-webui)
 
+## Getting started
 
-## Build Setup
+```bash
+# init submodule
+git submodule init
 
-``` bash
-# Install dependencies
-make install
+# update submodule
+git submodule init
+
+# install aw-client-js
+cd aw-client-js
+npm install
+cd ..
+
+# install aw-webui dependencies
+npm install
```

Comment:
Both the aw-client-js stuff and npm install can be run in one go with `make install`, but since `make` isn't commonplace on macOS or Windows maybe this is better.

Reply:
I'm using WSL and `make install` didn't seem to work. I maybe missed something.

Reply:
Ok, so it didn't work because I didn't have the submodule updated. I'll update to `make install` to simplify the instructions.